include/rdma/fi_ext.h

```diff
@@ -70,6 +70,7 @@ extern "C" {
 /* negative options are provider specific */
 enum {
        FI_OPT_EFA_RNR_RETRY = -FI_PROV_SPECIFIC_EFA,
+	   FI_OPT_EFA_SUPPORT_LL128
```

Comment:
Indention.  Make sure tab size=8

Reply:
Does changing this enumeration mean we need to bump some libfabric version number somewhere?

Reply:
> Does changing this enumeration mean we need to bump some libfabric version number somewhere?

This should be a backwardly compatible change (i.e. old code won't ask for this feature), so it should be safe to not bump the version #.  (Unless libfabric has a much stronger policy around interface changes like this)

Reply:
As we discussed previously, we need 2 options, one for send, another for RMA. Also do not use LL128, something like `EFA_SEND_DATA_IN_ORDER_ALIGNED_128_BYTE` and `EFA_RMA_DATA_IN_ORDER_ALIGNED_128_BYTE`